packages/task/src/browser/task-configurations.spec.ts

```diff
@@ -0,0 +1,174 @@
+/* eslint-disable */
```

Comment:
@allaVolkov why are you disabling `eslint` for the file? 
We should not disable the rule, and all errors and warnings present in the file should be addressed.

Reply:
removed